helm-charts/iqserver/templates/deployment.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         env:
           - name: SONATYPE_INTERNAL_HOST_SYSTEM
             value: Operator
+          - name: SONATYPE_WORK
+            value: {{ .Values.iq.configYaml.sonatypeWork }}
```

Comment:
Provide a SONATYPE_WORK env variable that will be used by default JAVA_OPTS from values.yaml/CSV. This is a fix found for the helm charts project.